catalogmanagementv1/catalog_management_v1_examples_test.go

```diff
@@ -0,0 +1,1728 @@
+// +build examples
+
+/**
+ * (C) Copyright IBM Corp. 2021.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package catalogmanagementv1_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/platform-services-go-sdk/catalogmanagementv1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+//
+// This file provides an example of how to use the Catalog Management service.
+//
+// The following configuration properties are assumed to be defined:
+// CATALOG_MANAGEMENT_URL=<service base url>
+// CATALOG_MANAGEMENT_AUTH_TYPE=iam
+// CATALOG_MANAGEMENT_APIKEY=<IAM apikey>
+// CATALOG_MANAGEMENT_AUTH_URL=<IAM token service base URL - omit this if using the production environment>
```

Comment:
I skipped this accidentally. Could you update this section with all the variables currently in use?

Reply:
Updated in all 4 SDK examples